inc/ocf_err.h

```diff
@@ -125,6 +125,9 @@ typedef enum {
 
 	/** Device does not meet requirements */
 	OCF_ERR_INVAL_CACHE_DEV,
+
+	/** Core device is in active state */
+	OCF_ERR_CORE_IN_ACTIVE_STATE,
```

Comment:
Is it going to be used anywhere? If so, it would be good to see both this commit and the one with actual usage in one PR so that we have a full picture

Reply:
Will be used in OCL. See PR mentioned in description.

Reply:
sorry, I missed that. but in this case this should be defined in OCL, there is a separate set of OCL error codes (i think "KCAS_ERR_...."